### DISCUSSION TITLE
.babelrc file is not applied.

### DISCUSSION THREAD

**Discussion by Dailyscat** [webpack/webpack#13609] [Q&A]
Good morning. First thanks for the help.
I want to use ts only as a form for type-check, so I use babel-loader and add only typescript-preset as shown below. It is not applied when writing the .babelrc file, but it is applied only if you give the option inside the babel-loader inside the webpack.config.js file like this. When I read the official documentation, I understood that babel-loader automatically recognizes and applies the .babelrc file. Am I wrong?

[myRepo](https://github.com/Dailyscat/lcp-common-window-dialog)

.babelrc
```
{
  "env": {
    "development": {
      "presets": ["@babel/preset-env"],
      "plugins": []
    },
    "production": {
      "presets": [
        [
          "@babel/preset-env",
          {
            "targets": { "ie": 11 }
          }
        ],
        "@babel/preset-typescript"
      ],
      "plugins": [["@babel/plugin-transform-runtime", { "corejs": 3 }]]
    }
  }
}

```

**Comment by alexander-akait** [comment]
Please ask this in `babel-loader` repo

**Reply by Dailyscat** [reply]
thanks!
